spec/requests/users_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
```

Comment:
- [ ] Good job on implementing all requested tests for the `users#index` action, Kindly also include all tests for the `users#show` action.